content/08-cardano-sidechains/03-example-evm-sidechain/09-discussion-faq.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Discussion & FAQ
+metaTitle:  Discussion & FAQ
+---
+
+This page answers some frequent questions for the Cardano Sidechains Toolkit and the proof of concept EVM sidechain application.
+
+## The Cardano Sidechains Toolkit
+
+These questions relate to the components IOG provides for building custom sidechains on Cardano.
+
+### Is the Sidechain Toolkit an open-source project?
+
+Yes, it will be open-sourced under the Input Output Global GitHub repository.
```

Comment:
Should be present tense, and should give the name of the repo (when we know it)

Reply:
New URL repo: https://github.com/input-output-hk/sc-evm @NeilBurgess42 @stevanlohja

Reply:
Committed suggested change. Please review and resolve this conversation if okay.